pythonsd/tests/__init__.py

```diff
@@ -16,6 +16,7 @@
 import webtest
 
 from config import wsgi
+from ..views import RecentVideosView
```

Comment:
I would combine these into a single import.
```suggestion
from ..views import RecentVideosView, UpcomingEventsView
```

Reply:
I actually prefer one import per line as it makes diffs simpler. Usually I use [reorder python imports](https://pypi.org/project/reorder-python-imports/) to enforce it.